pontoon/base/static/js/fluent_interface.js

```diff
@@ -23,6 +23,55 @@ var Pontoon = (function (my) {
       );
   }
 
+  function renderOriginalVariant(value, elements) {
+    return '<li>' +
+      '<span class="id">' +
+        value +
+      '</span>' +
+      '<span class="value">' +
+        Pontoon.fluent.serializePlaceables(elements, true) +
+      '</span>' +
+    '</li>';
+  }
+
+  /*
+   * Render original string elements.
+   *
+   * - Adjoining simple elements are concatenated and presented as a simple string
+   * - SelectExpression elements are presented as a list of variants
+   */
+  function renderOriginalElements(elements, title) {
+    var content = '';
+    var simpleElements = [];
+
+    elements.forEach(function (element, i) {
+      var isSimpleElement = Pontoon.fluent.isSimpleElement(element);
+      var isLastElement = i === elements.length - 1;
+
+      // Collect simple elements
+      if (isSimpleElement) {
+        simpleElements.push(element);
+      }
+
+      // Render collected simple elements when non-simple or last element is met
+      if (!isSimpleElement || isLastElement) {
+        if (simpleElements.length) {
+          content += renderOriginalVariant(title, simpleElements);
+          simpleElements = [];
+        }
+      }
+
+      // Render SelectExpression
+      if (element.type === 'SelectExpression') {
+        element.variants.forEach(function (item) {
```

Comment:
Forgot, not exactly sure about the context in which this is used, but do we want to find the default variant here instead of going over all?

Reply:
We need all variants here - this function is used in the source string panel.